.vib/spring-cloud-skipper-shell/goss/spring-cloud-skipper-shell.yaml

```diff
@@ -0,0 +1,12 @@
+command:
+  check-spring-cloud-skipper-shell:
+    timeout: 20000
+    exec: /opt/bitnami/java/bin/java -jar /opt/bitnami/spring-cloud-skipper-shell/spring-cloud-skipper-shell.jar --help
```

Comment:
Same feedback as in https://github.com/bitnami/containers/pull/34286/files#r1200343976